### DISCUSSION TITLE
pygame version to be used

### DISCUSSION THREAD

**Discussion by durkisneer1** [IndiePython/nodezator#49] [Suggestions]
I suggest moving from upstream pygame to the recently branched pygame-ce. It's a good investment to make in the long run as all the contributors moved from the former to the latter. You can view the new site at https://pyga.me/

**Comment by KennedyRichard** [comment]
Hello, Mr. Durk,

Just stopping by to acknowledge receipt of your message. I'll give you a proper reply in a couple of days or so (excluding Sundays).

I'll say this in advance, though: all things considered, I too think this is the best move. Nonetheless, I need to explain the situation to the Indie Python community first and give them the opportunity to share their feedback on the matter, specially after making a proper assessment of the whole situation.

Peace

**Comment by KennedyRichard** [comment]
Also, I'll take the liberty of converting your issue into a discussion.

Please, don't worry about it. This doesn't make your point any less important nor less visible (since our GitHub discussions are regularly used as well).

It is just that in the Indie Python project we reserve issues for reporting bugs that either crash the app or prevent an available feature to be used somehow.

This is for the benefit of all users, including your own. If you ever find a critical bug in Nodezator that needs urgent care, your reported issue won't ever need to compete for attention with other items that aren't nearly as urgent.

If you are curious you can read more about the reasoning behind this [here](https://github.com/IndiePython#issues-and-discussions).

In summary, this is just a way to help ensure issues are used for real issues rather than having the issues tab become a big and messy backlog or to-do list.

**Comment by KennedyRichard** [comment]
Hello, Mr. Durk,

Thank you for your patience.

Here's an announcement I made in response to your suggestion of switching to pygame-ce: https://github.com/orgs/IndiePython/discussions/2

Also, thank you for your suggestion and for reaching out to us.

Peace
